app/views/credits/new.html.erb

```diff
@@ -157,7 +157,7 @@
     var stripe = Stripe('<%= Settings::General.stripe_publishable_key %>');
     var elements = stripe.elements();
 
-    <% if current_user.decorate.dark_theme? %>
+    <% if current_user.dark_theme? %>
```

Comment:
We removed this method from the decorator, so this check now uses the predicate method defined by the enum option.